### PR TITLE
System 0.3.0: fix stream.yml.hbs files

### PR DESCRIPTION
**Pull request by fearful-symmetry**
This fixes the stream files that sends the config to the agent. See #78 for details.

Testing this is a tad more complicated than #78 as we need to make sure that the config that's getting sent to the agent is correct. The Fleet config manager has a YAML tab that you can use to view the raw config:

Screen Shot 2020-06-17 at 12 41 56 PM

However, you'll probably need to check the agent logs as well as the actual ingested data to make sure that metricset-level configs are working as intended.

### CODE REVIEW

File: packages/system/0.3.0/dataset/cpu/agent/stream/stream.yml.hbs [hunk 1]
```diff
@@ -1,17 +1,6 @@
...
-process.include_top_n.by_memory: {{process.include_top_n.by_memory}}
-{{/if}}
-{{#if processes}}
-processes: {{processes}}
-{{/if}}
+cpu.metrics:
+{{#each cpu.metrics}}
+  - {{this}}
+{{/each}}
+period: {{period}}
```
Comment (kaiyan-sheng) [inline packages/system/0.3.0/dataset/cpu/agent/stream/stream.yml.hbs:6]: New line at the end of the files?

**Comment by fearful-symmetry** [comment]
Latest commit removes the files that were causing issues with field mapping, and renames the dashboard. I also renamed a dashboard to fix that broken link. Unfortunately, adding the ECS fields fixes a mapping conflict but re-introduces the field mapping issue.

File: packages/system/0.3.0/dataset/cpu/fields/ecs.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      level: core
+      type: keyword
+      ignore_above: 1024
+      description: Host mac address.
+    - name: name
+      level: core
+      type: keyword
+      ignore_above: 1024
+      description: 'Name of the host.
+
```
Comment (kaiyan-sheng) [inline packages/system/0.3.0/dataset/cpu/fields/ecs.yml:31]: extra line?

File: packages/system/0.3.0/dataset/cpu/fields/ecs.yml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      level: extended
+      type: keyword
+      ignore_above: 1024
+      description: Operating system version as a raw string.
+      example: 10.14.1
+    - name: type
+      level: core
+      type: keyword
+      ignore_above: 1024
+      description: Type of host.
```
Comment (kaiyan-sheng) [inline packages/system/0.3.0/dataset/cpu/fields/ecs.yml:75]: needs extra line

**Pull request merged**